west.yml

```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: e2371d489f9aec8a3752cc945f8f81b19a7c2baf
+      revision: cd9a478213e0d5b42a73b68e0bab3eec3cd413e3
```

Comment:
Given our history of updating to newer Zephyr versions, even if this update is smooth and trivial and nothing relevant changed between those two commits, it still seems to me important enough for a separate PR